classes/musical_album.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  def initialize(publish_date, on_spotify: false)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
+
+  def to_s
+    %(On Spotify: #{@on_spotify} Published Year: #{@publish_date.year})
+  end
+
+  def to_json(json)
+    json.generate({ on_spotify: @on_spotify, publish_date: @publish_date.year })
+  end
+
+  private
+
+  def can_be_archived?
+    super && @on_spotify == true
+  end
```

Comment:
- [ ] according to the requirements `can_be_archived` should return true if the parent's method  returns `true` OR if `@on_spotify` equals true, in this case, it's only returning `true` if both cases are true

Reply:
Hi team, 
My sincere apologies but please ignore this comment I made a small mistake and mixed up the musical_album with another class